tests/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "running migrations"
+python3 manage.py migrate
+echo "creating superuser (admin/admin)"
+python3 manage.py createsuperuser
+echo "starting development server"
+python3 manage.py runserver 0.0.0.0:8000
+echo "EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'" > local_settings.py
+echo "launch SMTP debugging server"
+python -m smtpd -n -c DebuggingServer localhost:1025
```

Comment:
remove these two lines because they are not necessary anymore since we set django to send emails to the console output